proto/lbm/token/v1/query.proto

```diff
@@ -32,7 +32,7 @@ service Query {
   }
 
   // Grants queries grants on a given grantee.
-  rpc Grants(QueryGrantsRequest) returns (QueryGrantsResponse) {
+  rpc TokenGrants(QueryGrantsRequest) returns (QueryGrantsResponse) {
```

Comment:
I think this change is not a official change. Please check it @0Tech.

Reply:
> I think this change is not a official change. Please check it @0Tech.

I think he changed it to avoid some kind of error. I have encountered very same error before. At that moment, I changed my rpc name `Balances` into `TokenBalances`.
I think we may figure out how to resolve the error later.

Reply:
OK, I understand. Please check and correct in #412 later.